examples-next/basic/keystone.ts

```diff
@@ -22,7 +20,7 @@ const auth = createAuth({
   },
 });
 
-const isAccessAllowed = ({ session }: { session: any }) => !!session?.listKey;
+const isAccessAllowed = ({ session }: { session: any }) => !!session?.item?.isAdmin;
```

Comment:
For this not to explode, because the first user will be created in a way that can't access anything, you need to also set up auth config to specify `initFirstItem.extraCreateInput` (which was added for this purpose)